test/integration/cls.test.js

```diff
@@ -33,6 +33,16 @@ if (current.dialect.supports.transactions) {
     });
 
     describe('context', () => {
+      it('does not use continuation storage on manually managed transactions', function() {
+        const self = this;
+
+        return Sequelize._clsRun(() => {
+          return this.sequelize.transaction().then(() => {
```

Comment:
Remember to roll back the transaction, I think that's what's causing some of the other tests to hang

Reply:
Oooh, good point.  I'll fix that up right away.